Dockerfile.dispatcher

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:focal
```

Comment:
Not particularly, but it's working and I don't think it should make much difference since it's not actually building anything. Are you alright with leaving it?

Reply:
Good for me!